src/javascript/app_2/App/Components/Elements/PositionsDrawer/result-overlay.jsx

```diff
@@ -11,6 +11,7 @@ const ResultOverlay = ({
     onClick,
     onClickRemove,
     result,
+    display_shade,
```

Comment:
For `boolean` prop we could use `is_shade_visible` to make it consistent with other props in the project. `is_visible` is fine too, since this prop only applies within the context of this component.

Reply:
Sure,
Done.